server/databases.go

```diff
@@ -16,12 +16,12 @@ type dbLinks struct {
 }
 
 type dbResponse struct {
-	Name          string       `json:"name"`                        // a unique string identifier for the database
-	Duration      string       `json:"duration,omitempty"`          // the duration (when creating a default retention policy)
-	Replication   int32        `json:"replication,omitempty"`       // the replication factor (when creating a default retention policy)
-	ShardDuration string       `json:"shardDuration,omitempty"`     // the shard duration (when creating a default retention policy)
-	RPs           []rpResponse `json:"retentionPolicies,omitempty"` // RPs are the retention policies for a database
-	Links         dbLinks      `json:"links"`                       // Links are URI locations related to the database
+	Name          string       `json:"name"`                    // a unique string identifier for the database
+	Duration      string       `json:"duration,omitempty"`      // the duration (when creating a default retention policy)
+	Replication   int32        `json:"replication,omitempty"`   // the replication factor (when creating a default retention policy)
+	ShardDuration string       `json:"shardDuration,omitempty"` // the shard duration (when creating a default retention policy)
+	RPs           []rpResponse `json:"retentionPolicies"`       // RPs are the retention policies for a database
```

Comment:
It doesn't look like this is in `server/swagger.json` despite being here already (looks like it was introduced in 768bd12541c3b12561c7b75f2f4df62b771cd379). Would you mind adding it there?